### ISSUE TITLE
Image size

### ISSUE THREAD

**Issue by jijip41** [fac22/tiger-not-a-cat#28]
What would be the best way to decide a img size while we don't ruing page layout?

**Comment by danilo-cupido** [comment]
I've been thinking a lot about this issue and I've come to the conclusion that there is not much we can do without distorting the images. The problem is the API doesn't have a size property because if it had we could simply resize the photo since the height is over a certain limit (like we do with the width. 
The only viable solution would be to find an other API that do the same things (tags, filter, text).

**Comment by jijip41** [comment]
Yes I'm the same. I've tried to adjust img sizes in may ways but those were not really solving this issue. But I just thought of this which is we can make an image container with `<div>` and fix its size. Then make all images fit in this container as a background. We might need CSS for this:

background-position: center;
background-size: auto/length/cover/contain/initial/inherit;
background-repeat: no-repeat;
background-attachment: fixed;


These properties and values could be tested. If my theory works. 
But not sure about this cos we get all images using url from fetch... and we set a background image using the below property and the value. I cant' think of a way to use random img url as a background img.
Maybe there will be ways to do this using JS?

background-image: url(images/background-photo.jpg);
